packages/expo-doctor/src/api/getSchemaAsync.ts

```diff
@@ -0,0 +1,9 @@
+import fetch from 'node-fetch';
+
+export async function getSchemaAsync(sdkVersion: string): Promise<any> {
```

Comment:
A different function that uses sdkVersion - gteSdkVersion (https://github.com/expo/expo-cli/pull/4641/files#diff-6fdc6eae623e54996e6af13c27efa80c6d029dbfede1357467be1233471037b6R4) has a whole check in there for unversioned, here we just assume that `expoConfig.sdkVersion` is not unversioned and valid.